runtime/onert/core/src/exporter/train/CheckpointExporter.cc

```diff
@@ -59,14 +125,117 @@ class CheckpointExporter
       throw std::runtime_error{"Failed to save checkpoint: " + path};
 
```

Comment:
It may need verifier before writing file.

Reply:
I will add verifier for the buffer before writing a file later. Thanks.